src/content/case_studies/0002_em.md

```diff
@@ -0,0 +1,11 @@
+---
+title: 'Helping test algorithms'
+cover: '../../assets/bioimage-archive/case-study-1.gif'
+imageAlt: 'A resolution cryo-EM structure of beta-galactosidase in complex with a cell-permeant inhibitor'
+---
+
+Sjors Scheres, research leader at MRC Laboratory of Molecular Biology (LMB) and author of [RELION](https://www2.mrc-lmb.cam.ac.uk/relion/index.php?title=Main_Page), a widely used processing software for electron microscopy, was an early user of EMPIAR, both as submitter and reuser of data. 
+
+_"EMPIAR is very useful: it provides us with new data to test and develop our algorithms. Relying only on in-house data provides a very limited view but, because EMPIAR contains data from different sources, it allows us a much wider exposure to tune our algorithms."_, explains Scheres. _"Not having it would slow us down."_
+
+EMPIAR hosts the data used for RELION tutorials and benchmarks. Scheres was also involved in a EM community [map challenge](http://challenges.emdataresource.org/?q=2015_map_challenge), in which participants downloaded datasets from EMPIAR and processed them using different workflows and software to compare results. 
```

Comment:
Should we just remove the link completely? I can find https://www.emdataresource.org/news/announcing_map_challenge.html and a google result provides the same broken link as above. Otherwise could use: https://www.emdataresource.org/challenges/index.html?